records/generator.go

```diff
@@ -543,6 +553,66 @@ func (rg *RecordGenerator) insertRR(name, host string, kind rrsKind) (added bool
 	return
 }
 
+func rrsKindForIP(ip net.IP) rrsKind {
+	if ip.To4() != nil {
+		return A
+	} else if ip.To16() != nil {
+		return AAAA
+	}
+	return A
+}
+
+func rrsKindForIPStr(ip string) rrsKind {
+	if parsedIP := net.ParseIP(ip); parsedIP != nil {
+		return rrsKindForIP(parsedIP)
+	}
+	return A
```

Comment:
panic? otherwise, what are the cases where returning A makes sense here?

Reply:
See above comment.